src/module/item/affliction/data.ts

```diff
@@ -42,7 +42,7 @@ interface AfflictionOnset {
 
 interface AfflictionDamage {
     value: string;
-    type: DamageType;
+    damageType: DamageType;
```

Comment:
I liked `type` :(

Reply:
I agree, but everything from rule elements to melee damage uses damageType :(

Reply:
Converted back to type, we'll migrate everything else later.